compiler/src/iree/compiler/Codegen/SPIRV/SPIRVTileAndDistribute.cpp

```diff
@@ -136,8 +136,7 @@ class SPIRVTileAndDistributePass
 void SPIRVTileAndDistributePass::runOnOperation() {
   MLIRContext *context = &getContext();
   func::FuncOp funcOp = getOperation();
-  auto entryPointOp = getEntryPoint(funcOp);
-  if (!entryPointOp) return;
+  if (!isEntryPoint(funcOp)) return;
```

Comment:
do we need the check?

Reply:
maybe not, but dont want to change the semantics.